Duplicati/Library/Main/Operation/FilelistProcessor.cs

```diff
@@ -364,7 +418,15 @@ orderby n.Time descending
             }
 
             // cleanup deleted volumes in DB en block
-            database.RemoveRemoteVolumes(cleanupRemovedRemoteVolumes, null);
+            if (verifyMode == VerifyMode.VerifyStrict && missingUploadingVolumes.Count > 0)
+                throw new RemoteListVerificationException($"The remote volumes {string.Join(", ", missingUploadingVolumes)} are supposed to be deleted, but strict mode is on. Try running the \"repair\" command", "DeleteDuringStrictMode");
+            else if (verifyMode == VerifyMode.VerifyAndClean)
+            {
+                database.RemoveRemoteVolumes(missingUploadingVolumes.Concat(temporaryAndDeletedVolumes), null);
+                // Clear the flag after we have cleaned up
+                if (!options.Dryrun)
+                    database.TerminatedWithActiveUploads = false;
+            }

```

Comment:
Should a warning / message be emitted here with `VerifyOnly`?

Reply:
I think this follows the reasoning for for the other `VerifyOnly` that the operation is only interested to know if the data can be read safely. Partial stuff is not really important for listing for instance.